scripts/azure-gcp-wif/create-gcp-workload-identity-pool-provider.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Requires gcloud cli and authentication with gcloud:
+# - gcloud init
+# - gcloud auth login
+PROJECT="rugged-abacus-218110"
+
+# Create workload identity pool
+# NOTE: must be "global" region (Can't use for example "europe-west2")
+gcloud iam workload-identity-pools providers create-oidc azure-cip-oidc-provider \
+ --location="global" \
+ --workload-identity-pool="azure-cip-identity-pool" \
+ --issuer-uri="https://login.microsoftonline.com/9c7d9dd3-840c-4b3f-818e-552865082e16/v2.0" \
```

Comment:
9c7d9dd3-840c-4b3f-818e-552865082e16 is the azure tenant id. can you add it as a variable (like $PROJECT)